### PR TITLE
Add throttle config to UI

### PR DESCRIPTION
**Pull request by goodoldneon**
> :warning: Must release https://github.com/inngest/monorepo/pull/2602 before merging this PR

image

## Type of change (choose one)
- [ ] Chore (refactors, upgrades, etc.)
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] Security fix (non-breaking change that fixes a potential vulnerability)
- [ ] Docs
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
- [ ] I've linked any associated issues to this PR.
- [ ] I've tested my own changes.

*[Check our Pull Request Guidelines](https://github.com/inngest/inngest/blob/main/docs/PULL_REQUEST_GUIDELINES.md)*

### CODE REVIEW

File: ui/apps/dashboard/src/gql/graphql.ts [hunk 1]
```diff
@@ -1024,9 +1025,9 @@ export type RunsFilter = {
 };
 
 export type RunsFilterV2 = {
-  appID?: InputMaybe<Scalars['UUID']>;
+  appIDs?: InputMaybe<Array<Scalars['UUID']>>;
   from: Scalars['Time'];
-  functionID?: InputMaybe<Scalars['UUID']>;
+  functionIDs?: InputMaybe<Array<Scalars['UUID']>>;
   query?: InputMaybe<Scalars['String']>;
```
Comment (djfarrelly) [inline ui/apps/dashboard/src/gql/graphql.ts:1031]: Just flagging this unrelated change, I don't think it should cause any issues though

**Review by djfarrelly — APPROVED** [review]
:shipit:

**Review by anafilipadealmeida — APPROVED** [review]
